packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -24,7 +30,7 @@ jest.mock("../dialogs", () => {
  * then an error is thrown.
  */
 function mockConfirm(...expectations: { text: string; result: boolean }[]) {
-  (confirm as jest.Mock).mockImplementationOnce((text: string) => {
+  (confirm as jest.Mock).mockImplementation((text: string) => {
```

Comment:
We can pass multiple expectations to `mockConfirm()` so we should not remove the mock implementation after the first one.